python-ecosys/urequests/metadata.txt

```diff
@@ -1,4 +1,4 @@
 srctype = micropython-lib
 type = module
-version = 0.6
+version = 0.6.1
```

Comment:
This seems a more significant that a patch release.

Reply:
ah yep, this came in with one of the branches, I'll filter it back out into a new commit to update the overall version